scope

```diff
@@ -181,6 +187,7 @@ EOF
         if ! check_docker_for_mac || check_probe_only ; then
             check_not_running $SCOPE_CONTAINER_NAME $SCOPE_IMAGE_NAME
             docker rm -f $SCOPE_CONTAINER_NAME >/dev/null 2>&1 || true
+            create_plugins_dir
```

Comment:
It's a good idea to do this anyway, not just for Docker for Mac, as Docker has deprecated the behaviour where it creates new directory if path didn't exist.